packages/main/src/Card.js

```diff
@@ -36,6 +36,18 @@ const metadata = {
 			type: HTMLElement,
 			multiple: true,
 		},
+
+		/**
+		 * Defines the header of the <code>ui5-card</code>.
+		 * If not given standart card header will be applied.
+		 * @type {HTMLElement[]}
```

Comment:
- I guess the type should be @type {HTMLElement}, not HTMLElement[], as the slot is not multiple
- 'standart' should be 'standard'. 
- will be displayed, instead of 'will be applied', but this is a preference.